mod/uservalidationbyemail/views/default/resources/uservalidationbyemail/change_email.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+elgg_signed_request_gatekeeper();
+
+$user = elgg_call(ELGG_IGNORE_ACCESS | ELGG_SHOW_DISABLED_ENTITIES, function() {
+	return get_user((int) get_input('guid'));
+});
+
+if (!$user instanceof \ElggUser) {
+	throw new BadRequestException();
+}
+
+$shell = elgg_get_config('walled_garden') ? 'walled_garden' : 'default';
+
+$content = elgg_view('output/longtext', ['value' => elgg_echo('uservalidationbyemail:change_email:info')]);
+
+$content .= elgg_view_form('uservalidationbyemail/change_email', [], ['user' => $user]);
```

Comment:
I don't know if i agree with this. shouldn't the text be part of the form.

If someone wishes to reuse the form there is no explanation what it actually does....